packages/queries/src/types/QueryParam.ts

```diff
@@ -0,0 +1,5 @@
+export type QueryParam<TValue = unknown> = {
```

Comment:
The name `QueryParam` is generally associated with URL query parameters. To avoid this confusion I think we should call this `QueryParameter`